js/lib/ForceUnit.js

```diff
@@ -0,0 +1,218 @@
+};
+
+ForceUnit.prototype = new Measurement();
+ForceUnit.prototype.parent = Measurement;
+ForceUnit.prototype.constructor = ForceUnit;
+
+// https://en.wikipedia.org/wiki/Force
+ForceUnit.ratios = {
+    /*                        index dyne       mN          N           kN         MN         GN         kilogram-force pound-force poundal     */
+    "dyne":                  [ 1,   1,         1e-2,       1e-5,       1e-8,      1e-11,     1e-14,     1.01972e-6,    2.24809e-6, 7.23301e-5   ],
+    "millinewton":           [ 2,   100,       1,          1e-3,       1e-6,      1e-9,      1e-12,     1.01972e-4,    2.24809e-4, 7.23301e-3   ],
+    "newton":                [ 3,   1e5,       1000,       1,          1e-3,      1e-6,      1e-9,      0.101972,      0.224809,   7.23301      ],
+    "kilonewton":            [ 4,   1e8,       1e6,        1000,       1,         1e-3,      1e-6,      101.972,       224.809,    7.23301e3    ],
+    "meganewton":            [ 5,   1e11,      1e9,        1e6,        1000,      1,         1e-3,      1.01972e5,     2.24809e5,  7.23301e6    ],
+    "giganewton":            [ 6,   1e14,      1e12,       1e9,        1e6,       1000,      1,         1.01972e8,     2.24809e8,  7.23301e9    ],
+    "kilogram-force":        [ 7,   9.80665e5, 9.80665e-3, 9.80665,    9.80665e3, 9.80665e6, 9.80665e9, 1,             2.20462,    70.9316      ],
+    "pound-force":           [ 8,   4.44822e5, 4.44822e-3, 4.44822,    4.44822e3, 4.44822e6, 4.44822e9, 0.453592,      1,          32.174       ],
+    "poundal":               [ 9,   1.38255e4, 1.38255e-4, 1.38255e-1, 1.38255e2, 1.38255e5, 1.38255e8, 0.0140981,     0.031081,   1            ]
+};
+
+/**
+ * Return the type of this measurement. Examples are "mass",
+ * "length", "speed", etc. Measurements can only be converted
+ * to measurements of the same type.<p>
+ *
+ * The type of the units is determined automatically from the
+ * units. For example, the unit "grams" is type "mass". Use the
+ * static call {@link Measurement.getAvailableUnits}
+ * to find out what units this version of ilib supports.
+ *
+ * @return {string} the name of the type of this measurement
+ */
+ForceUnit.prototype.getMeasure = function() {
+    return "force";
+};
+
+/**
+ * Return a new instance of this type of measurement.
+ *
+ * @param {Object} params parameters to the constructor
+ * @return {Measurement} a measurement subclass instance
+ */
+ForceUnit.prototype.newUnit = function(params) {
+    return new ForceUnit(params);
+};
+
+ForceUnit.systems = {
+    "metric": [
+        "dyne",
+        "millinewton",
+        "newton",
+        "kilonewton",
+        "meganewton",
+        "giganewton"
+    ],
+    "uscustomary": [
+        "poundal",
+        "pound-force"
+    ],
+    "imperial": [
+        "poundal",
+        "pound-force"
+    ],
+    "conversions": {
+        "metric": {
+            "uscustomary": {
+                "millinewton": "pound-force",
+                "newton": "pound-force",
```

Comment:
Question) No mapping for poundal?

Reply:
The poundal is not used very often to measure any common things.